lighthouse-core/lib/i18n/locales.js

```diff
@@ -10,6 +10,7 @@
  *
  * Google locale inheritance rules: https://goto.google.com/ccssm
  * CLDR language aliases: https://www.unicode.org/cldr/charts/latest/supplemental/aliases.html
+ * CLDR locale inheritance: https://github.com/unicode-cldr/cldr-core/blob/master/supplemental/parentLocales.json
```

Comment:
wowza that's a lot of english ones, should we be doing something with those aliases here?

or is en-001 === en-US?

Reply:
> or is en-001 === en-US?

i initially was gonna say "yes", but apparently not!

first, the 419 in es-419 comes from [UN M49](https://en.wikipedia.org/wiki/UN_M49) and in there, 001 is world and 150 is europe.

BUT en-GB is also parented by en-001. (side note: en-150 seems pretty esoteric.. i think has more to do with date/time, but i'm not sure.)

see https://www.ctrl.blog/entry/en-001.html starting at **"The end of the US locale’s hegemony in CLDR"** 🤣 

http://cldr.unicode.org/development/development-process/design-proposals/english-inheritance helps even more because it spells out explicitly the en_XX that implicitly fall back to `en` which, yes, === `en_US`.

man this shit is fascinating.